### PR TITLE
Thomas default route

### PR DESCRIPTION
**Pull request by TKostrzewski**

### CODE REVIEW

File: client/src/pages/NoPageFound/NoPageFound.test.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { BrowserRouter } from 'react-router-dom';
+
+import NoPageFound from '.';
```
Comment (Joaoviana) [inline client/src/pages/NoPageFound/NoPageFound.test.jsx]: maybe keep it consistent with the text that it renders, such as: PageNotFound.jsx?

File: client/src/pages/NoPageFound/index.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+const NoPageFound = ({ location }) => {
+  const { pathname } = location;
+  return (
+    <Box display='flex' alignItems='center' justifyContent='center'>
+      <Box display='flex' flexDirection='column' alignItems='center'>
+        <Box mb={5}>
+          <Typography color='primary' variant='h5'>
+            404: Page not found. We&apos;re sorry, we couldn&apos;t find
+{' '}
```
Comment (Joaoviana) [inline client/src/pages/NoPageFound/index.jsx]: remove this unnecessary { ' '}

File: client/src/pages/NoPageFound/NoPageFound.test.jsx (hunk 1 above)
Comment (TKostrzewski) [reply client/src/pages/NoPageFound/NoPageFound.test.jsx]: Sounds good, will make that change

File: client/src/pages/NoPageFound/index.jsx (hunk 2 above)
Comment (TKostrzewski) [reply client/src/pages/NoPageFound/index.jsx]: I had issues with this. Some sort of conflict with the auto-formatting on save. I'd appreciate any and all help on this. I'll leave it in for this PR?

**Pull request merged**